src/app/cli/src/services/config/models.rs

```diff
@@ -606,6 +606,12 @@ impl DatabaseConfig {
             port: Some(DatabaseProvider::Postgres.default_port()),
         })
     }
+
+    pub fn sqlite_database() -> Self {
+        Self::Sqlite(SqliteDatabaseConfig {
+            database_path: "kamu.sqlite.db".into(),
```

Comment:
If we use this defualt path in CLI, it would place the file in the working directory.
I think in CLI mode we need to place the database within the `.kamu` workspace folder.

Reply:
Yes, fair comment -- I originally did that, but ran into issues and decided to postpone. 

Now I realize that this was the right direction and should be implemented that way. Will do

Reply:
Done, with commentary: the temporary directory is used for cases where we don't have a workspace yet, followed by a database move